go.mod

```diff
@@ -7,6 +7,7 @@ require (
 	github.com/ElrondNetwork/arwen-wasm-vm/v1_3 v1.3.30
 	github.com/ElrondNetwork/arwen-wasm-vm/v1_4 v1.4.8
 	github.com/ElrondNetwork/concurrent-map v0.1.3
+	github.com/ElrondNetwork/covalent-indexer-go v0.0.0-20210830114936-46b978c21f79
```

Comment:
proper release?

Reply:
Yes, this shall be changed when a proper release/development is ready. I've only added it to test it locally